models/biosample.py

```diff
@@ -0,0 +1,39 @@
+from dandischema import DandiBaseModel
+
+
+class BioSample(DandiBaseModel):
+    """Description of the sample that was studied"""
+
+    identifier: Identifier = Field(nskey="schema")
+    sampleType: SampleType = Field(
+        description="Identifier for the sample characteristics (e.g., from OBI, Encode).",
```

Comment:
I think 'characteristics' is too broad here -- better to have 'Identifier for the type of sample (e.g., from OBI, Encode).'